results/intfloat__multilingual-e5-small/ToxicChatClassification.json

```diff
@@ -1,15 +1,15 @@
 {
   "dataset_revision": "3e0319203c7162b9c9f8015b594441f979c199bc",
   "mteb_dataset_name": "ToxicChatClassification",
-  "mteb_version": "1.6.37",
+  "mteb_version": "1.7.14",
   "test": {
-    "accuracy": 0.710511562718991,
-    "accuracy_stderr": 0.07371659667449575,
-    "ap": 0.23112768741212197,
-    "ap_stderr": 0.03617857724708394,
-    "evaluation_time": 21.81,
-    "f1": 0.598941482258704,
-    "f1_stderr": 0.053856617238204875,
-    "main_score": 0.710511562718991
+    "accuracy": 0.7553264604810996,
```

Comment:
Slightly higher

Reply:
There is a problem with this task, it's a multilingual task but it doesn't inherit from `MultilingualTask`, I'm fixing this

Reply:
This task is a bit weird, it was added to the multilingual folder but it is not multilingual. At least the paper says that 99% of these chats are in English

Reply:
The [paper](https://arxiv.org/abs/2310.17389), and the [HF dataset link](https://huggingface.co/datasets/lmsys/toxic-chat). I just see English, can anyone confirm?

Reply:
I can confirm, I should have caught this at review time, my apologies!

Reply:
No worries, that's a bit hard to catch. Btw, multilingual datasets should inherit from `MultilingualTask`, so that we can choose on which language to evaluate.

Reply:
Thanks for taking a look at the score @imenelydiaker - It seems like there isn't any language tags so we can't add the language tag?

Reply:
I think we won't need language tags for this dataset since it's just in English, we'll move the task to the `eng` folder and fix the languages, I think @isaac-chung will open a PR to fix this, or I can do it here?

Reply:
If you can do it here, that'd be great. 🙏